app/views/collections/_form.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= simple_form_for @form, html: { class: 'editor' } do |f| %>
+  <div id="descriptions_display">
+
+  <div class="well">
```

Comment:
Can we move this below the collection_form_visibility partial?  Putting it first makes it feel like it is required.